anarchy-runner/files/ansible-runner/project/callback_plugins/anarchy.py

```diff
@@ -36,6 +36,16 @@
 def current_time():
     return '%sZ' % datetime.datetime.utcnow().isoformat()
 
+def munge_result(result):
+    """Return cleaned up and pruned version of result dict"""
+    # Clean up result for record
+    ret = result._result.copy()
+    if 'stdout' and 'stdout_lines' in ret:
+        del ret['stdout_lines']
+    if 'stderr' and 'stderr_lines' in ret:
+        del ret['stdout_lines']
+    return ret
```

Comment:
You're deleting `stdout_lines` in both conditions. You mean `stderr_lines` in the second one ?

Another version:

```python
def munge_result(result):
    ret = result._result.copy()
    ret.pop('stdout_lines', None)
    ret.pop('stderr_lines', None)
    return ret